<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CodecPreview</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="6,0,1">
      <item>
       <widget class="VideoPlaybackWidget" name="videoPlayback" native="true"/>
      </item>
      <item>
       <widget class="VideoStatisticsWidget" name="videoInfo" native="true"/>
      </item>
      <item>
       <widget class="QTabWidget" name="codecTabsWidget"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>711</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Input</string>
    </property>
    <addaction name="actionOpen_file"/>
    <addaction name="actionOpen_from_camera"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionCompare_multiple_codecs"/>
    <addaction name="actionMacroblocks"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionChange_raw_host"/>
    <addaction name="actionChange_raw_port"/>
    <addaction name="separator"/>
    <addaction name="actionChange_encoded_host"/>
    <addaction name="actionChange_encoded_port"/>
    <addaction name="separator"/>
    <addaction name="actionOriginal_size"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOpen_file">
   <property name="text">
    <string>Open from file</string>
   </property>
  </action>
  <action name="actionOpen_from_camera">
   <property name="text">
    <string>Open from camera</string>
   </property>
  </action>
  <action name="actionCompare_multiple_codecs">
   <property name="text">
    <string>Compare multiple codecs</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionChange_raw_port">
   <property name="text">
    <string>Change raw port</string>
   </property>
  </action>
  <action name="actionChange_encoded_port">
   <property name="text">
    <string>Change encoded port</string>
   </property>
  </action>
  <action name="actionChange_raw_host">
   <property name="text">
    <string>Change raw host</string>
   </property>
  </action>
  <action name="actionChange_encoded_host">
   <property name="text">
    <string>Change encoded host</string>
   </property>
  </action>
  <action name="actionMacroblocks">
   <property name="text">
    <string>Macroblocks</string>
   </property>
  </action>
  <action name="actionOriginal_size">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Original size</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>VideoStatisticsWidget</class>
   <extends>QWidget</extends>
   <header>VideoStatisticsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VideoPlaybackWidget</class>
   <extends>QWidget</extends>
   <header>videoplaybackwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
